app/components/primer/octicon_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render(Primer::BaseComponent.new(**@system_arguments)) do %>
+  <% if @use_symbol %>
+    <use href="#octicon-<%= [@icon.symbol, @icon.height].join("-") %>"></use>
```

Comment:
I wonder if there's performance here I can do by not looking up the `@icon` from cache or the gem, since I should have these 2 bits of data.

Reply:
I think I'll need to keep it the way it is, since the icon from octicons is still needed for some of the system_arguments. Maybe in the future, the gem just becomes a lookup gem.